model/src/main/kotlin/AdvisorRun.kt

```diff
@@ -52,4 +52,14 @@ data class AdvisorRun(
      * The result of this run.
      */
     val results: AdvisorRecord
-)
+) {
+    companion object {
+        val EMPTY = AdvisorRun(
+            startTime = Instant.EPOCH,
+            endTime = Instant.EPOCH,
+            environment = Environment(),
+            config = AdvisorConfiguration(),
```

Comment:
Unrelated, but I wonder whether we should rename these kind of constant to `DEFAULT` instead, as the values are not really empty.

Reply:
That's right, but the most relevant part is empty, like the results in this case. I guess there are good arguments for both names.